### PR TITLE
chore(VLayout/VFlex): add deprecate notice

### PR DESCRIPTION
**Pull request by MajesticPotatoe**
## Description
adds deprecation notice for `v-layout` / `v-flex`

## Motivation and Context
#8784 

## How Has This Been Tested?
visually

## Markup:

<details>

```vue
<template>
  <v-container>
    <v-layout>
      <v-flex></v-flex>
    </v-layout>
  </v-container>
</template>

<script>
  export default {
    data () {
      return {}
    },
  }
</script>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by KaelWD** [comment]
Do we actually want to force people off it already? Flooding the console with deprecation notices is a pretty strong motivator for something that's already opt-in with eslint.

**Comment by MajesticPotatoe** [comment]
> Do we actually want to force people off it already? Flooding the console with deprecation notices is a pretty strong motivator for something that's already opt-in with eslint.

it was suggested, but i could see if your whole app is still in 1.x how that wouldn't be very nice. could just shelve this for when we are ready.

File: packages/vuetify/src/components/VGrid/__tests__/VLayout.spec.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ describe('VLayout.ts', () => {
     }
   })
 
-  it('should work', () => {
+  it.skip('should work', () => {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VGrid/__tests__/VLayout.spec.ts]: `describe.skip` is better imho

**Comment by sh7dm** [comment]
personally I agree that we don't need to force people away from v1 grid

**Comment by sh7dm** [comment]
We'll reopen it when we need deprecation. do not remove branch

**Pull request closed without merging**